style.css

```diff
@@ -35,7 +36,7 @@ nav{
   text-decoration: none;
 }
 
-.container{
+.container {
   width: 100vw;
   height: 718px;
   background-image: url("Images/background.png");
```

Comment:
- [ ] Your container currently seems to have a width greater than the entire width of the screen. That is because you have a width of 100vw, which is the entire screen width and then you have some additional padding which makes it overflow the total size. That might be because you don't have `box-sizing: border-box`. This is a very cool and helpful concept which you can learn more about [here](https://www.w3schools.com/css/css3_box-sizing.asp). Kindly fix this error

- [ ] Your container does not have the round border at the left end corner which you can see below in the Figma template, Kindly add the round border at the end of your section.
> Yours
![image](https://user-images.githubusercontent.com/90556221/170800878-4bf659f3-6af6-4d98-8f6e-c402f10f6960.png)

> Figma
![image](https://user-images.githubusercontent.com/90556221/170800957-e47a2c81-2173-4473-9fa0-a0699f812661.png)